make-nagios-fhservices-cfg

```diff
@@ -24,6 +24,13 @@ rhmap_admin_email = os.getenv('RHMAP_ADMIN_EMAIL', 'root@localhost')
 rhmap_router_dns = os.getenv('RHMAP_ROUTER_DNS', 'localhost')
 rhmap_hostgroups = os.getenv('RHMAP_HOSTGROUPS', 'core')
 
+if rhmap_hostgroups == 'mbaas':
+    mongodb_count = int(os.getenv("MONGODB_ENDPOINT_COUNT", 3))
```

Comment:
We are going to set it.  It is in the template variables as ENDPOINT_COUNT, and only gets pulled into the environment variables for mongodb-initator, but assuming we move the mbaas to use configmaps like we do in core, then the nagios pod can automatically have access to that value.